src/browse.vue

```diff
@@ -595,6 +596,13 @@ export default {
         },
         resetQueryParams() {
             this.vuetableQueryParams.q = null
+        },
+        resetSortOrder() {
+            this.sortOrder = [{
+                field: "created_at",
+                sortField: "created_at",
```

Comment:
```suggestion
```
`sortField` is not required for `sortOrder`.

Reply:
It required after the second load or it will throw and error `[undefined]` is not a field.

Reply:
I see. Interesting. Thanks for the heads up. 👍🏻